Library/Homebrew/cli/args.rb

```diff
@@ -167,6 +180,19 @@ def devel
         (args_parsed && devel?) || cmdline_args.include?("--devel")
       end
 
+      def build_universal
+        (args_parsed && universal?) || cmdline_args.include?("--universal")
```

Comment:
Might be nice to split this (and all similar methods) into a `return` on each set of `()`

Reply:
I'm not sure I understand, could you please suggest the code?